ts/src/bybit.ts

```diff
@@ -7591,6 +7591,12 @@ export default class bybit extends Exchange {
         if (since !== undefined) {
             request['startTime'] = since;
         }
+        const until = this.safeInteger2 (params, 'until', 'till'); // unified in milliseconds
+        const endTime = this.safeInteger (params, 'endTime', until); // exchange-specific in milliseconds
```

Comment:
@sc0Vu we don't need to check the exchange-specific id here. If the user provides it, it will inevitably end up in the request because we extend params+request at the end.